### DISCUSSION TITLE
No Speed Sensor for 4090

### DISCUSSION THREAD

**Discussion by drdrumlord** [Rem0o/FanControl.Releases#2401] [Q&A]
No speed sensor was detected for my 4090 fans. When I select  No speed sensor and try to pair control to a speed sensor, none of the listed fans are affected. I also have seven fans listed and I have no idea what they are for. 
![Screenshot 2024-03-22 211645](https://github.com/Rem0o/FanControl.Releases/assets/29313659/908327b7-63b0-4be9-93ed-858d035642e6)

**Comment by ionGL** [comment]
Tick the manual control on both the 4090 fans controls, set them to 70-100%, and look at the speeds cards (not shown in your screenshot) and notice if any of the cards responds to your manual adjustments of the 4090 fans.
![manual](https://github.com/Rem0o/FanControl.Releases/assets/80151979/65a562a3-b63c-4c39-b95f-c45b7f458509)

**Reply by ionGL** [reply]
And make sure in sources you have nvapiwrapper ticked and gpu unticked on the left. Like this.
![nvapiwrapper](https://github.com/Rem0o/FanControl.Releases/assets/80151979/e9a5bf34-b42b-4400-abbf-8e316383d40b)

**Reply by drdrumlord** [reply]
Thank you. That helped me identify the proper fans. Will these two fans remain identified as "No Calibration"? 
![Screenshot 2024-03-23 055928](https://github.com/Rem0o/FanControl.Releases/assets/29313659/c28444b2-96fd-4d52-a790-770f7e3d4c29)

**Reply by ionGL** [reply]
First untick the "hide" option for the gpu controls, then select calibrate paired control automatically. 

You can later open the "edit calibration" for each gpu fan and fine tune things.

![calibrate](https://github.com/Rem0o/FanControl.Releases/assets/80151979/1be40ff7-375f-4dda-8e04-9c1ccbcb3c72)

**Comment by Rem0o** [comment]
If you click on the "no speed sensor" label" on your Nvidia tiles, there should be a dialog with a list of fans.   There should be fans named "nvidia fan 1" or something like that.

>  seven fans listed and I have no idea what they are for.

You are using the corsair link plugin right?  The Fan#1 to Fan#7 are your (empty) motherboard headers.

**Reply by drdrumlord** [reply]
Thank you!
